shared/editor/embeds/Castopod.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import Frame from "../components/Frame";
+import Image from "../components/Image";
+import { EmbedProps as Props } from ".";
+
+function Castopod(props: Props) {
```

Comment:
I don't think this makes sense as a first-class integration for a team knowledgebase to be honest, we have to maintain all of these so it doesn't come without cost.

Soon we'll support arbitrary iframes which will cover these kinds of cases.

Reply:
I understand your point. In that view, legitimacy of other integrations could also be challenged. For example, Spotify ?
Are you OK to integrate back Tldraw, or shall I just close this PR ?

Reply:
Ok with tldraw!